libs/cart/driver/magento/src/transforms/outputs/cart-item/simple-cart-item-transformer.ts

```diff
@@ -26,6 +26,10 @@ export function transformMagentoSimpleCartItem(cartItem: MagentoCartItem): DaffC
 			label: cartItem.product.thumbnail.label
 		},
 		in_stock: cartItem.product.stock_status === MagentoProductStockStatusEnum.InStock,
+    discounts: cartItem.prices.discounts.map(discount => ({
+      amount: discount.amount.value,
```

Comment:
probably could throw in a `discount.amount?.value` just in case

Reply:
changed